app/models/email.rb

```diff
@@ -26,35 +26,35 @@ def self.weekly_patrol_rosters(organisation = nil)
           next_roster = user_roster.first
           if user_roster.second.present?
             following_roster = user_roster.second
-            if user.club.is_active && user.club.enable_reminders_email
-              SwapseaMailer.weekly_roster_reminder(user, next_roster, following_roster, subject).deliver
+            if user.club.with_reminder_emails_enabled
+              SwapseaMailer.roster_reminder(user, next_roster, following_roster, subject).deliver
```

Comment:
Update to ".deliver_now" or ".deliver_later". Deliver later will make it async using ActiveJob.